code/modules/mob/language.dm

```diff
@@ -21,8 +21,8 @@
 		var/speaker_verb_override = speaker.get_spoken_verb(msg)
 		if (speaker_verb_override)
 			return speaker_verb_override
-	if(locate(/obj/item/clothing/head/cardborg) in speaker.get_equipped_items())
-		silicon = 1
+			if(locate(/obj/item/clothing/head/cardborg) in speaker.get_equipped_items())
```

Comment:
you overshot indent by one and now it will never trigger behind a return

Reply:
Oops. Thank you, fixed. I tested the runtime fix but didn't check if I accidentally ruined the helmet.